dockerfiles/Dockerfile.openvino

```diff
@@ -6,19 +6,16 @@
 FROM ubuntu:18.04
```

Comment:
Thanks. I tested this new Dockerfile and it seems to work well. Glad we don't have to manually download the software anymore.